bindgen/gen_odin.py

```diff
@@ -417,15 +405,25 @@ def gen_c_imports(inp, prefix):
     l(f'    when ODIN_DEBUG == true {{ foreign import {clib_import} {{ "{clib_prefix}_linux_x64_gl_debug.a"{linux_gl_libs} }} }}')
     l(f'    else                    {{ foreign import {clib_import} {{ "{clib_prefix}_linux_x64_gl_release.a"{linux_gl_libs} }} }}')
     l( '}')
-    l( '@(default_calling_convention="c")')
+
+    # Need to special case sapp_sg to avoid Odin's context keyword
+    if c_prefix == "sapp_sg":
+        l(f'@(default_calling_convention="c")')
+    else:
+        l(f'@(default_calling_convention="c", link_prefix="{c_prefix}")')
     l(f"foreign {clib_import} {{")
     prefix = inp['prefix']
     for decl in inp['decls']:
         if decl['kind'] == 'func' and not decl['is_dep'] and not check_ignore(decl['name']):
             args = funcdecl_args_c(decl, prefix)
             res_type = funcdecl_result_c(decl, prefix)
             res_str = '' if res_type == '' else f'-> {res_type}'
-            l(f"    {decl['name']} :: proc({args}) {res_str} ---")
+            # Need to special case sapp_sg to avoid Odin's context keyword
+            if c_prefix == "sapp_sg":
+                l(f'    @(link_name="{decl["name"]}")')
```

Comment:
Another option here would be to remove using `link_prefix` and then always use `@(link_name=...)` with `check_override(as_snake_case(...))` . This way there wouldn't need to be a special case for `sapp_sg`.